src/systems/sprites.js

```diff
@@ -47,22 +47,22 @@ AFRAME.registerComponent("sprite", {
   schema: { name: { type: "string" } },
   tick() {
     // TODO when we run out of sprites we currently just stop rendering them. We need to do something better.
-    if (!(this.didRegisterWithSystem || this.didFailToRegister) && this.el.sceneEl.systems["post-physics"]) {
-      const registered = this.el.sceneEl.systems["post-physics"].spriteSystem.add(this);
+    if (!(this.didRegisterWithSystem || this.didFailToRegister) && this.el.sceneEl.systems["hubs-systems"]) {
+      const registered = this.el.sceneEl.systems["hubs-systems"].spriteSystem.add(this);
```

Comment:
if you do refactor that `setInterval` business, I guess you'll need a `.ready` flag or something here.